ChaosMod/Util/Types.h

```diff
@@ -2,6 +2,21 @@
 
 #include "Natives.h"
 
+struct Vec2
+{
+	float x, y;
+};
+
+struct Vec3
+{
+	float x, y, z;
+};
```

Comment:
Might be better to rename them to `ChaosVector2` and `ChaosVector3` to differentiate them from the existing Vector type (which adds padding between each member and is intended for interoperating with natives).